go-controller/hack/update-codegen.sh

```
@@ -13,15 +13,15 @@ SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 cd "${builddir}"
-GO111MODULE=on go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.16.4
+GO111MODULE=on go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.19.0
 BINS=(
     deepcopy-gen
     applyconfiguration-gen
     client-gen
     informer-gen
     lister-gen
 )
-GO111MODULE=on go install $(printf "k8s.io/code-generator/cmd/%s@v0.32.3 " "${BINS[@]}")
+GO111MODULE=on go install $(printf "k8s.io/code-generator/cmd/%s@v0.34.1 " "${BINS[@]}")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)